backend/api/views.py

```diff
@@ -1,3 +1,19 @@
-from django.shortcuts import render
+from rest_framework import viewsets, mixins
+from serializers import BusinessSerializer
+
+from sodapy import Socrata
+
+class BusinessesViewSet(mixins.ListModelMixin,
+                        viewsets.GenericViewSet):
+
+    serializer_class = BusinessSerializer
+
+    def get_queryset(self):
+        client = Socrata("data.cityofnewyork.us", "z9bHXnSMLOVBFTGrELvWCcCBN")
+        businesses = client.get("9w7m-hzhe", content_type="json",
+                                limit="500000",
+                                select="camis, dba, boro, building, street, zipcode, cuisine_description")
+        # Get all unique businesses
+        businesses = {v['camis']:v for v in businesses}.values()
+        return businesses
```

Comment:
Be careful of what you're returning here. Whatever you return needs to be compatible with your serializer class, so you might need to add some logic here that will ensure compatibility (to whatever extent you can or need to ensure that).